russian/buildings-dep/call-mum/call-mum.md

```diff
@@ -0,0 +1,2 @@
+Вы машинально набираете мамин номер, но вместо длинных гудков слышите голос робота:
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.